4.3.0 grammar review stage 2 (#5020)
wazuh/wazuh-documentation: source/upgrade-guide/legacy/upgrading-elastic-stack/from-2.x-to-5.x.rst

Upgrading Elastic Stack from 2.x to 5.x
=======================================

> Although Wazuh 2.x is compatible with both Elastic Stack 2.x and 5.x, it is recommended that version 5.x be installed since the Wazuh Kibana plugin is not compatible with Elastic Stack 2.x.

Reviewer: ```suggestion
Although Wazuh 2.x is compatible with both Elastic Stack 2.x and 5.x, it is recommended to install version 5.x since the Wazuh Kibana plugin is not compatible with Elastic Stack 2.x.
```

#. Remove deprecated settings from the Elasticsearch configuration files:

>     Removing deprecated settings on Elasticsearch will avoid errors and conflicts after the upgrade. To do this, add the following lines in the ``/etc/elasticsearch/elasticsearch.yml`` configuration file:

Reviewer: ```suggestion
    Removing deprecated settings on Elasticsearch will avoid errors and conflicts after the upgrade. To do this, comment the following lines in the ``/etc/elasticsearch/elasticsearch.yml`` configuration file:
```
